src/OpenTracing/Mock/MockSpan.php

```diff
@@ -13,7 +13,7 @@ final class MockSpan implements Span
     private $operationName;
 
     /**
-     * @var MockSpanContext
+     * @var SpanContext
```

Comment:
Since this is internal type, does it make sense to keep the specific instead of the generic?

Reply:
I tried to get rid of this phpstan message:

     ------ ---------------------------------------------------- 
      Line   OpenTracing/Mock/MockSpan.php                       
     ------ ---------------------------------------------------- 
      133    Property OpenTracing\Mock\MockSpan::$context        
             (OpenTracing\Mock\MockSpanContext) does not accept  
             OpenTracing\SpanContext|null.                       
     ------ ----------------------------------------------------

Do you have a better idea?

Reply:
I just checked narrowing 

    public function withBaggageItem(string $key, string $value): ?SpanContext

in `MockSpanContext` to

    public function withBaggageItem(string $key, string $value): ?MockSpanContext

but this is only allowed since PHP 7.4